src/index.css

```diff
@@ -7,12 +7,12 @@
 * {
   margin: 0;
   padding: 0;
-  font: inherit;
+  /*font: inherit;*/
   /* Get rid of all font sizes and heights */
 }
 
 html {
-  color-scheme: dark light;
+  /*color-scheme: dark light;*/
```

Comment:
Let's delete this rather that just a comment, which might make people think it should be added back in.

Reply:
Same for above with the font property.